[Bug] - Make sure change.value is a string before checking for specific keywords

```js
foundry.js:655 
 Error: Error thrown in hooked function '' for hook 'preCreateActiveEffect'. change.value?.includes is not a function
[Detected 1 package: bugs(12.444.1)]
//...
Caused by: TypeError: change.value?.includes is not a function
[Detected 1 package: bugs(12.444.1)]
    at bugs-main.mjs:499:46
```

--- a/Changelog.md
+++ b/Changelog.md
@@ -1,3 +1,7 @@
+## v13.346.1
+* Compatibility bump
+* Fix for some conditions not populating changes
+
 ## v12.439.5
 * Added handling for rolling `1dc` instead of `1d2`, if Dice so Nice module is active and only 2 available effects to choose from for `[autoMidiChooseEffects]`.
 * If Dice so Nice module is not active, a message with the roll's details will be created to give some sense to what's happening.

--- a/module.json
+++ b/module.json
@@ -2,7 +2,7 @@
     "id": "bugs",
     "title": "Bugbear's Scripts",
     "description": "A small module for Foundry and Dnd5e which contains scripts thislonelybugbear finds useful.",
-    "version": "12.444.1",
+    "version": "13.346.1",
     "authors": [
         {
             "name": "thatlonelybugbear",
@@ -14,7 +14,7 @@
     "compatibility": {
         "minimum": "12",
         "maximum": "13",
-        "verified": "12.343"
+        "verified": "13.346"
     },
     "esmodules": ["scripts/bugs-main.mjs"],
     "relationships": {
@@ -24,8 +24,8 @@
                 "type": "system",
                 "compatibility": {
                     "minimum": "4.3.0",
-                    "maximum": "4.4.9",
-                    "verified": "4.4.4"
+                    "maximum": "5.1",
+                    "verified": "5.0.4"
                 }
             }
         ],
@@ -64,6 +64,6 @@
     "url": "https://github.com/thatlonelybugbear/bugs",
     "license": "https://raw.githubusercontent.com/thatlonelybugbear/bugs/main/LICENSE",
     "manifest": "https://github.com/thatlonelybugbear/bugs/releases/latest/download/module.json",
-    "download": "https://github.com/thatlonelybugbear/bugs/releases/download/v12.444.1/module.zip",
+    "download": "https://github.com/thatlonelybugbear/bugs/releases/download/v13.346.1/module.zip",
     "changelog": "https://raw.githubusercontent.com/thatlonelybugbear/bugs/main/Changelog.md"
 }

--- a/scripts/bugs-main.mjs
+++ b/scripts/bugs-main.mjs
@@ -494,6 +494,7 @@ Hooks.once('midi-qol.ready', () => {
 			const changes = getChanges(ae);
 			if (!changes || foundry.utils.isEmpty(changes)) return true;
 			changes.filter((change) => {
+				if (typeof(change.value) !== 'string') return false;
 				const hasOriginTokenUuid = change.value.includes('originTokenUuid');
 				const hasOriginTokenId = change.value.includes('originTokenId');
 				const hasOriginToken = change.value.includes('originToken');

